daemon/containerd/service.go

```diff
@@ -74,19 +76,25 @@ func (cs *ImageService) Images(ctx context.Context, opts types.ImageListOptions)
 	}
 
 	var ret []*types.ImageSummary
+	snapshotter := cs.client.SnapshotService(containerd.DefaultSnapshotter)
```

Comment:
> DefaultSnapshotter

Will this be configurable in a follow-up PR?

LGTM then

Reply:
Yes, we should make this configurable; looks like https://github.com/rumpl/moby is currently only using the default (but that's hard-coded in containerd to `overlayfs` for Linux), and we should make that configurable, similar to the existing graph drivers (and have feature detection, likely, if we also want automatic selection)

./cc @rumpl

Reply:
Indeed we haven't yet added this but we definitely need it, there are some docker commands that will need to have a `--snapshotter` flag.